tests/tools/sgx-lkl-docker/Makefile

```diff
@@ -0,0 +1,53 @@
+include ../../common.mk
+
+APP_ROOT=app
+PROG=${APP_ROOT}/python-helloworld.py
+
+DISK_IMAGE=rootfs.img
+DATA_IMAGE=data.img
+IMAGE_SIZE=100M
+
+# The re-packaged Docker image.
+WRAP_DOCKER_IMAGE=test-sgx-lkl-docker
+WRAP_DOCKER_IMAGE_STAMP=$(WRAP_DOCKER_IMAGE).stamp
+# Note that an arbitrary/non-existing non-root user is used for launching the container.
```

Comment:
Why is this?

Reply:
Just as a proof that it doesn't just work with the pre-defined non-root user inside the image. It probably makes sense to remove the pre-defined user (999) altogether. I created that user before I added 777 permissions to the copied disk images. Once I remove the pre-defined user then I can just `USER 1000` and remove the comment above. What do you think?

Reply:
I did the change, works fine.